modules/local/changeo/changeo_buildtrees.nf

```diff
@@ -2,7 +2,7 @@ process CHANGEO_BUILDTREES {
     tag "$meta.id"
     label 'process_medium'
 
-    conda (params.enable_conda ? "conda-forge::r-base=4.1.2 bioconda:r-alakazam=1.2.0 bioconda::changeo=1.2.0 bioconda::igphyml=1.1.3" : null)              // Conda package
```

Comment:
In most files there is still the `              // Conda package`, not sure if that is intended.

Reply:
thanks, also removed the other ones now